src/community/script/core/src/test/java/org/geoserver/script/rest/SessionTest.java

```diff
@@ -7,16 +7,13 @@
 
 import net.sf.json.JSONArray;
 import net.sf.json.JSONObject;
-
 import org.geoserver.script.ScriptIntTestSupport;
-import org.geoserver.test.GeoServerTestSupport;
-
 import org.springframework.mock.web.MockHttpServletResponse;
 
 public class SessionTest extends ScriptIntTestSupport {
 
     public void testPost() throws Exception {
-        MockHttpServletResponse response = postAsServletResponse("/script/sessions/js", "");
+        MockHttpServletResponse response = postAsServletResponse("/rest/sessions/js", "");
```

Comment:
Likewise, this should probably be `/rest/script/sessions`, and the sessions controller should be updated.

The restlet framework handled servlet paths differently from the MVC framework I believe.
Either way, you should verify in a 2.11 geoserver with script installed to be sure of the correct path.

Reply:
I tested in 2.9.x, will test with 2.11 if you think that would be a better test.

Reply:
2.9 is fine.

Reply:
Tested with 2.9.x and this is the expected behavior and the correct path.

Reply:
From the Restlet versions applicationContext, the mapping for SessionFinder, which is now SessionController is as follows.
```
<!-- Scripting Sessions -->
        <entry>
          <key><value>/sessions</value></key>
          <value>sessionFinder</value>
        </entry>
        <entry>
          <key><value>/sessions/{ext}</value></key>
          <value>sessionFinder</value>
        </entry>
        <entry>
          <key><value>/sessions/{ext}/{session}</value></key>
          <value>sessionFinder</value>
        </entry>
```
Therefore this is the correct path.

Reply:
After some investigation, it turns out the scripts extension proxies the REST api at /script, such that both `"/script/sessions/js"` and `"/rest/sessions/js"` are valid paths:

```
<bean id="scriptDispatcherMapping" 
        class="org.springframework.web.servlet.handler.SimpleUrlHandlerMapping">
    <property name="alwaysUseFullPath" value="true"/>
    <property name="mappings">
        <props>
            <prop key="/script/**">restWrapper</prop>
        </props>
    </property>
  </bean>
```

All the controllers will need to be updated to support both paths.